### DISCUSSION TITLE
[SOLVED] Saving Live Config choices

### DISCUSSION THREAD

**Discussion by AM8431** [end-4/dots-hyprland#792] [Q&A] [answered]
In the Live Config section in the right sidebar, I can't figure out how to save certain options. For example, I would like to save the option that prevents Arch from sleeping, and I would also like to know how to save the blur size. I can't find anything in the config files that could save those options for me, so any help is appreciated!

**Comment by end-4** [comment]
the live config is there because i thought "hey ughh this might looks cool imma make it yes yes"

for hyprland options, edit hyprland configs
etc

**Comment by end-4** [comment, marked as answer]
prevent fron sleeping? edit ~/.config/hyprland/hypridle.conf to your liking. set timeout to 696969696969 or something

**Reply by AM8431** [reply]
That fixed my issues, thanks for the help!
